prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -819,7 +819,8 @@ class ActionHistory : Logging {
                     "never - skipSend specified"
                 }
                 reportFile.nextActionAt != null -> {
-                    "${reportFile.nextActionAt}"
+                    JacksonMapperUtilities.timestampFormatter
```

Comment:
Note: Note that soon this JSON generation will be replaced by a class and using Jackson.